### PR TITLE
feat: adapt Qt6

### PR DESCRIPTION
**Pull request by starhcq**
adapt Qt6

Log: adapt Qt6

### CODE REVIEW

**Comment by pengfeixx** [comment]
[是否满足兼容性要求] Y
[是否满足commit提交规范] Y
[是否满足编码规范] Y
[Review结论] Pass
[Fail原因] N/A

File: src/src/utils/devicehelper.h [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    bool isExist(const QString &deviceId);
+
+    // 判断url是否为smb网络路径
+    static bool isSamba(const QUrl &url);
+private:
+    static DeviceHelper *m_instance;
+
+    QStringList getProtocalDeviceIds();
+private:
+    QScopedPointer<QDBusInterface> dfmDeviceManager; // 文管设备管理DBus服务接口
```
Comment (lzwind) [inline src/src/utils/devicehelper.h:58]: 变量是否合理

File: src/src/utils/devicehelper.cpp [hunk 2]
```diff
@@ -0,0 +1,227 @@
...
+void DeviceHelper::loadAllDeviceInfos()
+{
+    m_mapDevicesInfos.clear();
+    QStringList blockDeviceIds = getBlockDeviceIds();
+    QStringList protocalDeviceIds = getProtocalDeviceIds();
+    QStringList deviceIds;
+    deviceIds << blockDeviceIds << protocalDeviceIds;
+    QMap<QString, QVariantMap> deviceInfos;
+    for (auto id : deviceIds) {
+        QVariantMap infos = loadDeviceInfo(id, true);
```
Comment (lzwind) [inline src/src/utils/devicehelper.cpp:92]: 去掉infos

**Review by lzwind — APPROVED** [review]

**Review by lzwind — CHANGES_REQUESTED** [review]
需要调整后在review

File: src/src/utils/devicehelper.h (hunk 1 above)
Comment (starhcq) [reply src/src/utils/devicehelper.h:58]: 已修改

File: src/src/utils/devicehelper.cpp (hunk 2 above)
Comment (starhcq) [reply src/src/utils/devicehelper.cpp:92]: 已去掉

**Review by starhcq — COMMENTED** [review]
done

**Comment by starhcq** [comment]
> 需要调整后在review

已调整

**Review by lzwind — APPROVED** [review]

**Pull request merged**